### ISSUE TITLE
Problems with git credentials creating new VCS backed project

### ISSUE THREAD

**Issue by mud5150** [OctopusDeploy/cli#222]
## The bug

I'm attempting to create a new project in Octopus via the cli. I'm using an existing git credential stored in the library. When I run the command without the `--git-credential-store` argument I'm prompted for this information, even though the docs state "Library" is the default. When I supply `--git-credential-store "Library"` I'm prompted for the git username, which I shouldn't need to supply since it should be coming from the library value. It seems like perhaps it's trying to create an new library value. I want to utilize an existing library credential. 

## Command to reproduce

```
octopus project create --process-vcs -n ${PWD##*/} -l <lifecycle_name> -g "Default Project Group" --git-url $(git config --get remote.origin.url) --git-credentials <library_credential_name> --git-credential-store "Library" -s Default
```

## Outcome

```
? The Git username.
? Git username 
```

## Versions

**cli**: 1.2.0
P.S. you should add a version flag

**Octopus Server**: 2023.1.8263-hotfix.8481

**Comment by mud5150** [comment]
Hmm, ok. That's unexpected. I added `--no-prompt` and it worked without supplying the additional arguments. I was then also able to remove `--git-credential-store "Library"` as well. I'm glad I was able to move forward, but that seems like strange behavior to continue to prompt for values that aren't needed. Maybe the expectations around how those arguments are supposed to work together just isn't clear. IDK
